ci-operator/step-registry/layered-product/opp/apply-opp-policyset/layered-product-opp-apply-opp-policyset-commands.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+set -x
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+# create the policies namespace
+oc apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: policies
+EOF
+
+# apply the configure subscription admin policy
+oc apply -f - <<EOF
```

Comment:
Using an ACM community policy to do this can be beneficial for testing the policy and for keeping this code up to date without needing maintain this specific bash script.

The policy to use would be https://github.com/stolostron/policy-collection/blob/main/community/CM-Configuration-Management/policy-aws-machine-sets.yaml

Reply:
This policy is statically assigning 1 replica per machineset, which means we can only create 3 nodes.

Also this requires that we have 3 regions a,b, and c